### DISCUSSION TITLE
음원 및 영상 질문

### DISCUSSION THREAD

**Discussion by diddntjd99** [naverz/zepeto-studio-kor#95] [5. Q&A]
1. 영상질문 - #90 질문을 남겼었는데, 영상이 아닌 컷툰처럼 카메라 무빙을 통해 보여지는 것은 괜찮나요?

2. 음원 질문 - 무료 음원 출처는 description에 남기라 나와있는데, 월드 생성할 때 적을 수 있는 설명칸을 지징하는 것인가요?

**Comment by beanyh** [comment]
1. 영상 포함이 안되기 때문에, 이미지를 연속하여 발표자료처럼 만들어 재생한 사례가 있습니다. 이미지를 활용하시면 좋을듯 합니다. 
2. 네~ '월드 설명' 에 현재로는 기입 부탁드립니다.
